pkg/licensing/classifier.go

```diff
@@ -67,20 +56,12 @@ func Classify(r io.Reader) ([]types.LicenseFinding, error) {
 	return findings, nil
 }
 
-// FullClassify uses two classifiers to detect and classify the license found in a file
-func FullClassify(filePath string, contents []byte) (types.LicenseFile, error) {
+// GoogleClassify uses Google classifier to detect and classify the license found in a file
```

Comment:
We can just remove this function and call the existing `Classify` instead.

Reply:
One second. I might be wrong.

Reply:
ok, i'll try to remove it

Reply:
I found out those two functions returned different values.

Reply:
I'll think of that again tomorrow.